src/pkg/artifact/model.go

```diff
@@ -31,7 +31,7 @@ import (
 // for all users.
 type Artifact struct {
 	ID                int64                  `json:"id"`
-	Type              string                 `json:"type"`                // image, chart, etc
```

Comment:
Pls keep the previous comments, the artifact is used for oci registry, not the chartmuseum artifact,  the type could be chart if it is a chart with OCI artifact.

Reply:
fixed